modules/oe_theme_helper/templates/oe-theme-helper-in-page-navigation.html.twig

```diff
@@ -44,6 +46,9 @@
   {% endblock %}
   {% block content %}
   <div class="ecl-col-lg-9">
+    {% if content_prefix %}
```

Comment:
Let's use blocks here, together with Twig's `parent()` functionality. Let's move the blocks within the wrappers:

![image](https://user-images.githubusercontent.com/153362/110093415-fdb70e00-7d9a-11eb-8908-8ed61e072c0c.png)

This change will have an impact on templates that extend this, such as the one for the organisation content type. Fix that too.